tests/scalers/prometheus.test.ts

```diff
@@ -160,7 +153,6 @@ spec:
   cooldownPeriod:  10
   triggers:
   - type: prometheus
-    metricType: AverageValue
```

Comment:
+1

Reply:
@zroubalik I removed the `Metric type should be "AverageValue"` test from the default case, since the metric type in the ScaledObject spec will be empty and not `AverageValue`. Maybe we should check the HPA metric target type instead, because this will always be be set in `GetMetricSpecForScaling()`.

Reply:
yeah, you are right :)